docs/translations.md

```diff
@@ -1,43 +1,777 @@
+development. The front end uses [frontend/pendingTranslations.ftl][]. The back end code
+uses [privaterelay/pending_locales/en/pending.ftl][].
+
+New approved strings go through several steps to get translated. The developers open a
+pull request to the translation repository. The Localization team reviews the new
+strings. When the Localization team merges the changes, Pontoon imports the new strings.
+The language teams translate the strings. Pontoon exports them to the translation
+repository. The nightly action updates the submodule to pull in the latest translations.
+The release Docker image includes the new translations. Once released, the users see the
+content in their language.
+
+["en" bundle]: https://github.com/mozilla-l10n/fx-private-relay-l10n/tree/main/en
+[Django back end]: https://github.com/mozilla/fx-private-relay/blob/main/privaterelay/ftl_bundles.py
+[FAQ page]: https://relay.firefox.com/faq/
+[Firefox Accounts]: https://accounts.firefox.com/
+[Firefox Relay Add-on]: https://addons.mozilla.org/firefox/addon/private-relay/
+[Firefox Relay Website]: https://relay.firefox.com/
+[Fluent format]: https://projectfluent.org/
+[GitHub action]: https://github.com/mozilla/fx-private-relay/actions/workflows/l10n-sync.yml
+[Mozilla Localization Team]: https://github.com/mozilla-l10n
+[addons.mozilla.org]: https://addons.mozilla.org/
+[faq.ftl]: https://github.com/mozilla-l10n/fx-private-relay-l10n/blob/main/en/faq.ftl
+[git submodules]: https://git-scm.com/book/en/v2/Git-Tools-Submodules
+[mozilla-l10n/fx-private-relay-add-on-l10n]: https://github.com/mozilla-l10n/fx-private-relay-add-on-l10n
+[mozilla-l10n/sumo-l10n]: https://github.com/mozilla-l10n/sumo-l10n
+[mozilla-l10n/www-l10n]: https://github.com/mozilla-l10n/www-l10n
+[mozilla.org]: https://www.mozilla.org/
+[mozilla/addons-frontend]: https://github.com/mozilla/addons-frontend/locale
+[mozilla/fxa-content-server-l10n]: https://github.com/mozilla/fxa-content-server-l10n
+[privaterelay/locales]: https://github.com/mozilla/fx-private-relay/tree/main/privaterelay
+[subscription pages]: https://subscriptions.firefox.com/subscriptions
+[support.mozilla.org]: https://support.mozilla.org/
+
+## The Technical Details of Region Selection
+
+Relay premium plans are not available world-wide, only in some regions. As of September
+2023, the premium email service is available in 34 regions. Phone infrastructure varies,
+so the phone service in available in only 2 regions. This section details the
+technologies used to distinguish users by region.
+
+### Identifying the User's Region
+
+The stage and production deployments use similar load balancers. These set a
+`X-Client-Region` header based on the visitor's IP address. There is no way for a user to
+override this region selection. A tester can use a [VPN][] to change their IP address
+and their detected region.
+
+The `X-Client-Region` header is sometimes missing. In stage and production, some IP
```

Comment:
(question, non-blocking) Is this the header set by Google Cloud Platform (GCP) that shows up on requests because our service is running on GCP? So if we for some reason change our infrastructure away from GCP this will fail.